monocle/webapp.py

```diff
@@ -121,11 +121,37 @@ def whoami():
     return jsonify(username)
 
 
-@app.route("/api/0/query/<name>", methods=["GET"])
-def query(name):
+def _get_index(request):
     if not request.args.get("index"):
         abort(make_response(jsonify(errors=["No index provided"]), 404))
-    index = request.args.get("index")
+    return request.args.get("index")
+
+
+def _get_project_definition(index):
+    monocle_config = yaml.safe_load(open(config_path)) if config_path else None
+    return (
+        config.get_project_definition(monocle_config, index) if monocle_config else None
+    )
+
+
+@app.route("/api/0/config_project_def", methods=["GET"])
+def get_cfg_project_definition():
+    index = _get_index(request)
+    projects = _get_project_definition(index)
+    if not projects:
```

Comment:
If no project definition then simply return an empty list

Reply:
Then in react will complicate next pull request, where axios will get array with promise object that it will comes to undefined and it will other issues. Can not be like it is now?

Reply:
ok